<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pswd_forgetw</class>
 <widget class="QWidget" name="pswd_forgetw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nameless Bank</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #bdbdbd;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Nameless&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLineEdit" name="le_username">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	background: #E0E0E0;
	border: 2px solid #666666;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #F9F9F9;
	border-right-color: #F9F9F9;
	color: #000000;
 }
QLineEdit:hover {
	background: #E0E0E0;
	border: 2px solid #666666;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #F9F9F9;
	border-right-color: #F9F9F9;
	color: #000000;
 }</string>
         </property>
         <property name="placeholderText">
          <string>Имя пользователя:</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_control_question">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Контрольный вопрос:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_control_question2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_control_question">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	background: #E0E0E0;
	border: 2px solid #666666;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #F9F9F9;
	border-right-color: #F9F9F9;
	color: #000000;
 }
QLineEdit:hover {
	background: #E0E0E0;
	border: 2px solid #666666;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #F9F9F9;
	border-right-color: #F9F9F9;
	color: #000000;
 }</string>
           </property>
           <property name="placeholderText">
            <string>Введите ответ на контрольный вопрос:</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btn_restore">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background: #cdcdcd;
	border: 2px solid #F9F9F9;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #111111;
	border-right-color: #111111;
	color: #111111;
 }
QPushButton:pressed {
	background: #aaaaaa;
	border: 2px solid #111111;
	border-bottom-width: 1px;
	border-right-width: 1px;
	border-bottom-color: #F9F9F9;
	border-right-color: #F9F9F9;
	color: #333333;
 }
QPushButton:hover {
	background: #aaaaaa;
	color: #333333;
 }</string>
         </property>
         <property name="text">
          <string>Восстановить пароль</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lbl_pswd">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Пароль: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
